src/components/PostInfo/PostInfo.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import UserInfo from '../UserInfo/index';
+import CommentList from '../CommentList/index';
+
+type Props = {
+  posts: Post;
+};
+
+export const PostInfo: React.FC<Props> = ({ posts }) => {
+  const {
+    title, body, user, comment,
+  } = posts;
```

Comment:
Here types are okay, but still i'd suggest destructuring in function body
```suggestion
type Props = {
  posts: Post;
};

export const PostInfo: React.FC<Props> = (props) => {
  const { posts } = props;
  const {
    title, body, user, comment,
  } = posts;
```